### PR TITLE
issue_41 new build script to pull all SIS_import/*.rb into a tar file

### PR DESCRIPTION
**Pull request by zqian**
please review the new shell script to generate tar file with SIS_import/*.rb files.

### CODE REVIEW

File: sis_import_build.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/sh --
+
+## find the next Jenkins build number
+## in a file named nextBuildNumber two parent directory up
+parentdir="$(dirname "$(pwd)")"
```
Comment (dlhaines) [inline sis_import_build.sh]: Jenkins provides this in an environment variable: BUILD_NUMBER

The build script should work on a desktop too.  You won't have the BUILD_NUMBER variable then but you can default it to something like "local_build". The syntax is VARIABLE=${1:-DEFAULTVALUE}

File: sis_import_build.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/sh --
```
Comment (dlhaines) [inline sis_import_build.sh]: You and Pushyami should have the build scripts on the same level.  IMHO the second level directory with the files for just one project would be a good place.

File: sis_import_build.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                mkdir artifact
+                chmod -R 755 artifact
+        else
+                ## exist artifact
+                echo "artifact Folder exist"
+        fi
+        find . -maxdepth 1 -name "*.rb" -exec tar -rf ./artifact/SIS_import_$build_number.tar {} \;
+else
+        # no file named nextBuildNumber
+        echo "File $FILE does not exists"
```
Comment (dlhaines) [inline sis_import_build.sh]: In this case it should return a failing exit code so it is easier to know that the build failed.

File: sis_import_build.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        ## move to the folder with SIS scripts
+        cd SIS_import
+        ## check the artifact folder, whether it exists or not
+        if [ ! -d "artifact" ]; then
+                ## no artifact folder
+                ## create such folder first
+                echo "create artifact folder"
+                mkdir artifact
+                chmod -R 755 artifact
+        else
```
Comment (dlhaines) [inline sis_import_build.sh]: Not sure this else clause adds much.

File: sis_import_build.sh (hunk 2 above)
Comment (pushyamig) [inline sis_import_build.sh]: very soon after the production deploy of sections project we are going to move the sections tool out of canvas.

File: sis_import_build.sh (hunk 1 above)
Comment (zqian) [inline sis_import_build.sh]: script updated.

File: sis_import_build.sh (hunk 2 above)
Comment (zqian) [inline sis_import_build.sh]: script moved into the SIS_import folder

File: sis_import_build.sh (hunk 3 above)
Comment (zqian) [inline sis_import_build.sh]: the if-else structure is removed after adapting the approach to read build number from env variable.

File: sis_import_build.sh (hunk 4 above)
Comment (zqian) [inline sis_import_build.sh]: removed else section

**Pull request merged**